### PR TITLE
Update to latest Avalonia

### PR DESCRIPTION
**Pull request by jp2masa**
## Changes
- Update to latest Avalonia.

### CODE REVIEW

File: src/Avalonia.Xaml.Interactivity/Interaction.cs [hunk 1]
```diff
@@ -31,15 +31,15 @@ static Interaction()
 
                 if (newCollection != null && e.Sender != null)
                 {
-                    newCollection.Attach(e.Sender);
+                    newCollection.Attach(e.Sender as AvaloniaObject ?? throw new InvalidOperationException());
```
Comment (jp2masa) [inline src/Avalonia.Xaml.Interactivity/Interaction.cs]: The only other option is breaking a lot of public APIs (interfaces).

**Comment by wieslawsoltes** [comment]
@jp2masa I am not planning on using Avalonia nightly builds for this as they can not be used for official NuGet releases.

**Comment by jp2masa** [comment]
@wieslawsoltes I understand, but my problem is that I can't use this library with Avalonia nightly builds, as there were some breaking changes. Maybe there could be nightly builds 0.9.999-... or would that be too much work?

**Comment by wieslawsoltes** [comment]
> @wieslawsoltes I understand, but my problem is that I can't use this library with Avalonia nightly builds, as there were some breaking changes. Maybe there could be nightly builds 0.9.999-... or would that be too much work?

AvaloniaBehaviors has nightly builds, but it takes time to keep-up with Avalonia master branch and breaking changes.

**Comment by wieslawsoltes** [comment]
@jp2masa I was trying to keep up with nightly Avalonia builds but I have many repositories and syncing all of them and dependencies with nightly feed takes also time and is not really practical https://github.com/wieslawsoltes?utf8=%E2%9C%93&tab=repositories&q=Avalonia&type=source&language=

**Comment by wieslawsoltes** [comment]
Having master  branch in sync with Avalonia master also requires maintaining stable branches like 0.9.X as Avalonia releases cherry-picked changes from master to stable branches like 0.9.X.

**Comment by jp2masa** [comment]
I understand, I'll close the PR then.

**Pull request closed without merging**